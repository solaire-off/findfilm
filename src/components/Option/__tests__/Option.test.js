import React from "react";
import { render } from "@testing-library/react";
import { Option } from "..";

const label = "Some option";

it("Current render when check prop", () => {
  const { asFragment } = render(
    <Option
      label={label}
      isSelected={true}
      getStyles={() => {}}
      cx={() => {}}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});

it("Current render when uncheck prop", () => {
  const { asFragment } = render(
    <Option
      label={label}
      isSelected={false}
      getStyles={() => {}}
      cx={() => {}}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
