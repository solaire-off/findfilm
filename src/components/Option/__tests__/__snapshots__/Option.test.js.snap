// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current render when check prop 1`] = `
<DocumentFragment>
  <div>
    <div
      class="css-o8533i-Option"
    >
      <input
        checked=""
        type="checkbox"
      />
      <label>
        Some option
      </label>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Current render when uncheck prop 1`] = `
<DocumentFragment>
  <div>
    <div
      class="css-o8533i-Option"
    >
      <input
        type="checkbox"
      />
      <label>
        Some option
      </label>
    </div>
  </div>
</DocumentFragment>
`;
